src/Clapi/Relay.hs

```diff
@@ -92,6 +93,25 @@ genInitDigest ps tns vs =
         RtnDataSeries ts ->
           d'{ocdData = alInsert p (oppifyTimeSeries ts) $ ocdData d}
 
+contDiff
+  :: AssocList Seg (Maybe Attributee) -> AssocList Seg (Maybe Attributee)
+  -> Map Seg (Maybe Attributee, SequenceOp Seg)
+contDiff a b = afters <> absents
+  where
+    asAfter (acc, prev) k ma = ((k, (ma, SoPresentAfter prev)) : acc, Just k)
+    aftersFor = Map.fromList . fst . alFoldlWithKey asAfter ([], Nothing)
+    aftersA = aftersFor a
+    aftersB = aftersFor b
+    afters = Map.difference aftersB aftersA
```

Comment:
Maybe do a merge for the afters/absents thingy

Reply:
We had a thing at one point that did the kind of "diffs in each direction" thing, which was a little simpler than a full-blown merge...